dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/importer/context/ProgramInstanceSupplier.java

```diff
@@ -151,7 +151,7 @@ private void mapExistingEventsToProgramInstances( ImportOptions importOptions, L
         // Collect all the Program Instances by event uid
         final Map<String, ProgramInstance> programInstancesByEvent = getProgramInstanceByEvent( importOptions, events );
```

Comment:
Same here as in another place, I prefer we ensure that `programInstancesByEvent` is never null. This makes the code safer and easier to read. We are too defensive 😬